tower-http/src/catch_panic.rs

```diff
@@ -341,7 +349,7 @@ impl ResponseForPanic for DefaultResponseForPanic {
             );
         };
 
-        let mut res = Response::new(Full::from("Service panicked"));
+        let mut res = Response::new(Full::new(http_body_util::Full::from("Service panicked")));
```

Comment:
You anyway make use of it 🤷 Might as well just re-rexpose it then if you really want to ? Not sure I see the value. I imagine it has to do with stability guarantees, but dunno... Would hold stronger the argument if you do not link into it at all.

Reply:
What jplatte where here is the reason https://github.com/tower-rs/tower-http/pull/348#discussion_r1399789699

Using stuff from http-body-util internally is fine but we won't expose those types in our public API.

Reply:
Agreed.